### PR TITLE
fix: prevent account resetting on issue

### PR DESCRIPTION
**Pull request by nud3l**

### CODE REVIEW

File: src/pages/app/app.page.tsx [hunk 1]
```diff
@@ -8,9 +8,7 @@ import RedeemSteps from './redeem/redeem-steps';
 import RedeemRequests from './redeem/redeem-requests';
 import Transfer from './transfer/transfer';
 import { setActiveTabAction } from '../../common/actions/general.actions';
-import { FaArrowLeft } from 'react-icons/fa';
 import './app.page.scss';
-import { changeIssueStepAction } from '../../common/actions/issue.actions';
 import TestnetBanner from '../../common/components/testnet-banner';
```
Comment (anton-karlovskiy) [inline src/pages/app/app.page.tsx:12]: Friendly tip: we could use absolute import paths (super helpful :)).
E.g. import { setActiveTabAction } from 'common/actions/general.actions';

File: src/pages/app/app.page.tsx [hunk 2]
```diff
@@ -51,12 +38,6 @@ export default function AppPage(): ReactElement {
             }>
             {hideTabs() ? (
               <React.Fragment>
```
Comment (anton-karlovskiy) [inline src/pages/app/app.page.tsx:40]: FYI: Now we could use `<></>` instead of `<React.Fragment></React.Fragment>`.

**Comment by anton-karlovskiy** [comment]
Overall LGTM!
Just left a few minor comments. They are nice to do.

File: src/pages/app/app.page.tsx (hunk 1 above)
Comment (nud3l) [reply src/pages/app/app.page.tsx:12]: agree, will be fixed in future PRs!

File: src/pages/app/app.page.tsx (hunk 2 above)
Comment (nud3l) [reply src/pages/app/app.page.tsx:40]: Maybe we could just do a search-replace in one big PR to do that?

**Review by anton-karlovskiy — APPROVED** [review]

**Pull request merged**